Examples/FireBaseUI/FireBaseUIViewController.swift

```diff
@@ -55,6 +74,13 @@ internal final class FireBaseUIViewController: ViewController {
   var btnReset = Button(frame: Rect(x: 256, y: 324, width: 156, height: 32),
                         title: "Reset Password")
 
+  lazy var fetchUserDocumentButton: Button = {
```

Comment:
I think that the `lazy` can be elided if we just initialise the value.

Reply:
I can't reference other value in the initializer which I'd like to be able to do in order to do some relative layout.

Reply:
Bleh, I completely missed that!